assets/src/components/notifications.tsx

```diff
@@ -6,10 +6,17 @@ import { Notification, NotificationReason } from "../realtime.d"
 import { formattedTimeDiff, now } from "../util/dateTime"
 import PropertiesList from "./propertiesList"
 
+const fullstoryEvent = (): void => {
+  if (window.FS && window.username) {
```

Comment:
I expect we're also going to be doing a lot of this `if (window.FS && ...` in the near future, do you think we should extract a helper now or is that maybe premature?

Reply:
Let's wait to see how the next use of fullstory works to see what would be a way to DRY this.